modules/server/src/network/index.ts

```diff
@@ -108,14 +115,31 @@ const fetchRemoteSchemas = async ({
 	return schemaResults;
 };
 
+/**
+ * Connects to remote network connections, introspects their GQL schemas and merges to output single schema
+ * @param { networkConfigs }
+ * @returns graphql schema for the network - types and resolvers combined
+ */
 export const createSchemaFromNetworkConfig = async ({
 	networkConfigs,
 }: {
 	networkConfigs: NetworkAggregationConfigInput[];
-}): Promise<NetworkAggregationConfig[]> => {
-	const remoteSchemasResult = await fetchRemoteSchemas({
+}) => {
+	const networkConfigsWithSchemas = await fetchRemoteSchemas({
 		networkConfigs,
 	});
 
-	return remoteSchemasResult;
+	// filter remote connections with valid schemas
+	const gqlSchemas = networkConfigsWithSchemas
+		.filter((config) => config.schema !== null)
+		.map((config) => config.schema as GraphQLSchema);
```

Comment:
can change this to a filter func if you'd like, I feel there's a cleaner way to do this overall.